<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_red">
            <property name="text">
             <string>RED</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="spin_red">
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QSlider" name="vslider_red">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_green">
            <property name="text">
             <string>GREEN</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="spin_green">
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QSlider" name="vslider_green">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_blue">
            <property name="text">
             <string>BLUE</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="spin_blue">
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QSlider" name="vslider_blue">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>vslider_red</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_red</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_red</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vslider_red</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vslider_green</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_green</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_green</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vslider_green</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vslider_blue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_blue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_blue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vslider_blue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
